src/UserList.js

```diff
@@ -1,34 +1,87 @@
 import React from 'react';
-import usersData from './data/users.json';
-import User from './components/user.js';
+import User from './components/User.js';
 import './css/App.css';
+import UserStore from './stores/UserStore';
+import UserAction from './actions/UserAction';
 
 
 export default class UserList extends React.Component {
     constructor (props) {
         super(props);
 
-        this.state = {
-            wantAdd: false
-        };
+        this.state = this.getAppState();
 
         this.handleClickAddUser = this.handleClickAddUser.bind(this);
     }
 
+    getAppState () {
+        return {
+            wantAdd: false,
+            users:   UserStore.getUsers()
+        };
+    }
+
     handleClickAddUser () {
         this.setState({ wantAdd: !this.state.wantAdd });
     }
 
+    handleClickSubmit () {
```

Comment:
This method looks bad, we should rewrite it in the next PR.
In sudh case we add comment in the code, e.g.
`//TODO: should be rewriten`
1) I think we should have `NewUser` or `NewUserForm` component. This component should incupsulate `User` and `Submit` button. Now it isn't good that `UserList` is trying to  manipulate with its child's dom structure.
2) It's better to use `refs` to get elements' values https://ru.reactjs.org/docs/refs-and-the-dom.html
or implement `onChange` handlers to accumulate entered values.
3) this form will use store's action directly

Please add a comment and create a new issue in your repo.